pkg/lint/lintersdb/manager.go

```diff
@@ -245,22 +257,14 @@ func (m *Manager) combineGoAnalysisLinters(linters map[string]*linter.Config) {
 		return a.Name() <= b.Name()
 	})
 
-	ml := goanalysis.NewMetaLinter(goanalysisLinters)
+	mlConfig.Linter = goanalysis.NewMetaLinter(goanalysisLinters)
```

Comment:
Is there a reason this is set after the linters loop like this instead of in the creating of `mlConfig` on L207?

```go
mlConfig := &linter.Config{
    Linter: goanalysis.NewMetaLinter(goanalysisLinters),
}
```

Feels more natural to me but maybe something is preventing this in `ConsiderSlow`?

Reply:
The linters loop is on a map: due to alt names, linters can be duplicated (they are deduplicated by using `.Names()`).
 
The computation of `LoadMode` and `IsSlow` is easier and more readable by not using local variables.

I will rewrite the `presets` computation to follow the same "style" as `LoadMode` and `IsSlow`.